src/artemis/experiment_plans/tests/test_fast_grid_scan_plan.py

```diff
@@ -308,6 +308,7 @@ def test_when_grid_scan_ran_then_eiger_disarmed_before_zocalo_end(
     fake_fgs_composite.eiger.odin.check_odin_state = MagicMock(return_value=True)
     fake_fgs_composite.eiger.odin.file_writer.num_captured.sim_put(1200)
     fake_fgs_composite.eiger.stage = MagicMock()
+    fake_fgs_composite.eiger.stage.return_value = Status(None, None, 0, True, True)
```

Comment:
Nit: You can do this in one line
```suggestion
    fake_fgs_composite.eiger.stage = MagicMock(return_value=Status(None, None, 0, True, True))
```

Reply:
still doesn't fit on one line with our linter ;)